docs/using/selecting.md

```diff
@@ -264,5 +264,9 @@ core images and link them below.
   `base-notebook` image.
   Try it on [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/knonm/kotlin-notebook/main)
 
+- [transformers_notebook](https://github.com/ToluClassics/transformers_notebook): Easily run JupyterLab Notebooks pre-installed with the
```

Comment:
This link doesn't work. Could you fix it please?

Reply:
Oh yeah true; it was a private repo but now it's public

Reply:
Thanks!